src/solver/model_getters.jl

```diff
@@ -357,13 +360,26 @@ function get_microphysics_model(parsed_args)
     end
 end
 
-function get_cloud_model(parsed_args)
+function get_cloud_model(parsed_args, params)
     cloud_model = parsed_args["cloud_model"]
     FT = parsed_args["FLOAT_TYPE"] == "Float64" ? Float64 : Float32
     return if cloud_model == "grid_scale"
         GridScaleCloud()
     elseif cloud_model == "quadrature"
         QuadratureCloud(SGSQuadrature(FT))
+    elseif cloud_model == "cloud_ml"
+        nn_filepath = joinpath(
+            @clima_artifact("cloud_fraction_nn"),
+            "arch_2layers_8nodes.jld2",
```

Comment:
could consider moving the strings `"cloud_fraction_nn"` and `"arch_2layers_8nodes.jld2"` to the config/toml file so that you can easily change them later as you refine the model.